k8s/gitlab/staging/release.yaml

```diff
@@ -71,7 +71,7 @@ data:
     - op: remove
       path: /spec/values/gitlab/gitlab-shell/service
     - op: replace
-      path: /spec/valuesFrom/name
+      path: /spec/valuesFrom/0/name
```

Comment:
I'm happy to just approve this and see what happens, but is there a way to test the patch syntax locally to make sure it's correct?

Reply:
yes, you can run the json patching script manually with
`python3 scripts/gitops-patch.py k8s/gitlab/release.yaml k8s/gitlab/staging/release.yaml`

Reply:
Maybe it would be worth having a CI check that runs that script on every staging patch

Reply:
Yeah, that might make sense.